README.md

```diff
@@ -237,7 +237,6 @@ module "tgw" {
 * [Complete example with static routes](https://github.com/terraform-aws-modules/terraform-aws-vpn-gateway/tree/master/examples/complete-vpn-gateway-with-static-routes) shows how to create all VPN Gateway together with static routes.
 * [Minimal example](https://github.com/terraform-aws-modules/terraform-aws-vpn-gateway/tree/master/examples/minimal-vpn-gateway) shows how to create just VPN Gateway using this module.
 
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Please revert these removals in all README files since they are required for autogenerated documentation (by `pre-commit-terraform` hook).

Reply:
I will fix the failing Github Actions once you revert these changes. Sorry for the confusion with this.

Reply:
No Worries Anton, reverted docs change just now :)